src/Windows/R/Components/Impl/Resources.resx

```diff
@@ -442,12 +442,18 @@ If you want to delete all data on the remote machine you must delete your user p
   <data name="ConnectionManager_RemoveConnectionConfirmation" xml:space="preserve">
     <value>Are you sure you want to remove the '{0}' connection?</value>
   </data>
+  <data name="ContainerManager_ContainerServiceIsNotInstalled" xml:space="preserve">
+    <value>Docker service isn't installed.\nTo start working with containers, install Docker for Windows and restart VS.</value>
```

Comment:
Does `\n` actually work here? since it is `xml:space="preserve"` it should be just regular formatting

Reply:
Good catch.